scripts/spread-hunter-menu.ps1

```
@@ -0,0 +1,1936 @@
+    when under the repo, home-relative (~/...) otherwise. Paths that are
+    already relative pass through unchanged. #>
+    param([string]$Path)
+    if ([string]::IsNullOrWhiteSpace($Path)) { return $Path }
+    $norm = $Path -replace '\\', '/'
+    if ($norm -notmatch '^[A-Za-z]:/') { return $norm }
+    $proj = $ProjectPath -replace '\\', '/'
+    if ($norm.StartsWith($proj + '/', [System.StringComparison]::OrdinalIgnoreCase)) {
+        return $norm.Substring($proj.Length + 1)
+    }
+    if ($env:USERPROFILE) {
+        $home = $env:USERPROFILE -replace '\\', '/'
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'home' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)